contracts/truefi2/TrueFiCreditOracle.sol

```diff
@@ -131,6 +131,7 @@ contract TrueFiCreditOracle is ITrueFiCreditOracle, UpgradeableClaimable {
      * @dev Set new manager for updating scores
      */
     function setManager(address newManager) public onlyOwner {
+        require(address(newManager) != address(0), "TrueFiCreditOracle: Cannot set new manager to address(0)");
```

Comment:
Why do we need to cast `newManager` to `address`, while it's already an `address`?

Reply:
Thanks for catching that!